nss-day-cohort-60/national-parks-api
# Client can now request all and a single campground(s), camping reservation(s), event(s), and event registration(s)

# Description 
Created four new classes for campground, camping reservation, event, and event registration.  Created four new requests modules for campground, camping reservation, event, and event registration and added two new functions to each module.  The first function is get_all() which returns all of the rows from each table.  The second function is get_single() which returns a single row from each table based on the id provided by the client.

# Steps to test get_all()

1. Fetch the vs-campground-views branch and switch to it
2. Start/restart debugger
3. Send a GET request for all campground data (e.g. http://localhost:8088/campgrounds)
4. Verify that 200 status code and the entire list of dictionaries are received
5. Repeat process for the following GET requests:
     - http://localhost:8088/events
     - http://localhost:8088/event_registration
     - http://localhost:8088/camping_reservations

# Steps to test get_single()

1. Fetch the vs-campground-views branch and switch to it
2. Start/restart debugger
3. Send a GET request for an existing campground (e.g. http://localhost:8088/campgrounds/1)
4. Verify that 200 status code and the single dictionary with id = 1 are received
5. Repeat process for the following GET requests:
     - http://localhost:8088/events/1
     - http://localhost:8088/event_registration/1
     - http://localhost:8088/camping_reservations/1
